README.md

```diff
@@ -43,16 +43,16 @@ We are using Weblate as a translation platform. [Register on Weblate](https://ho
 ### Writing plugins
 
 Mumble supports general-purpose plugins that can provide functionality that is not implemented in the main Mumble application. You can find more
-information on how this works and on how these have to be created in the [plugin documentation](docs/dev/plugins/README.md).
+information on how this works and how these have to be created in the [plugin documentation](docs/dev/plugins/README.md).
```

Comment:
I think the "on" in the original version is important :thinking:

Reply:
The "on" is redundant and can be removed.
![image](https://github.com/mumble-voip/mumble/assets/2989985/336c6f9b-6784-4b41-b517-9827f15dd4ae)

Or to put it syntactically:
```
… information on [how this works and how these have to be created] in the …
```
equals
```
… information on [this matter] in the …
```

Reply:
> I think the "on" in the original version is important :thinking: 

Not really, it is just taking extra space there